### PR TITLE
Splitter: create unit and testcafe testing to test new added icons

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts [hunk 1]
```diff
@@ -78,19 +78,27 @@ class ResizeHandle extends (Widget as any) {
...
+        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} ${this._getResizeIconClass()}`;
       default:
-        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} dx-icon-overflow`;
+        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} .dx-icon-overflow`;
     }
   }
 
+  _getResizeIconClass(): string {
+    const isHorizontal = this._isHorizontalDirection();
+    console.log(isHorizontal);
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]: Seems, I forgot to remove the console.

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts [hunk 2]
```diff
@@ -78,19 +78,27 @@ class ResizeHandle extends (Widget as any) {
...
+    
+    return isHorizontal ? 'dx-icon-handlevertical' : 'dx-icon-handlehorizontal';
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]:
```suggestion
    return `dx-icon-handle${ isHorizontal ? 'vertical' : 'horizontal' };
```

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts [hunk 3]
```diff
@@ -78,19 +78,27 @@ class ResizeHandle extends (Widget as any) {
       case 'next':
         return `${RESIZE_HANDLE_COLLAPSE_NEXT_PANE_CLASS} ${ICON_CLASS} ${this._getCollapseIconClass(true)}`;
       case 'icon':
+        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} ${this._getResizeIconClass()}`;
       default:
-        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} dx-icon-overflow`;
+        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} .dx-icon-overflow`;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]: We can type incoming iconType argument. It should accept 'prev', 'icon', 'next' values. After this we can remove processing inside `default` case.

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]: ok sir will change this one

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts (hunk 3 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]: correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/6754eeced384460fb394c2160b6e5d628e935f80)

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]: correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/6754eeced384460fb394c2160b6e5d628e935f80)

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]: correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/6754eeced384460fb394c2160b6e5d628e935f80)

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        text: 'pane_2',
+      },
+      ],
+    });
+  });
+
+  test('Nested Splitter icon test', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, `Nested Splitter appearance, orientation(${orientation}).png`, { shouldTestInCompact: true });
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]:
```suggestion
    await testScreenshot(t, takeScreenshot, `Nested Splitter appearance, orientation='${orientation}'.png`, { shouldTestInCompact: true });
```

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+import { testScreenshot } from '../../../helpers/themeUtils';
+
+fixture.disablePageReloads`Splitter_keyboard`
+  .page(url(__dirname, '../../container.html'));
+
+['horizontal', 'vertical'].forEach((orientation) => {
+  test('Splitter icon test', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, `Splitter appearance, orientation(${orientation}).png`, { shouldTestInCompact: true });
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]:
```suggestion
    await testScreenshot(t, takeScreenshot, `Splitter appearance, orientation='${orientation}'.png`, { shouldTestInCompact: true });
```

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import { createWidget } from '../../../helpers/createWidget';
+import { testScreenshot } from '../../../helpers/themeUtils';
+
+fixture.disablePageReloads`Splitter_keyboard`
+  .page(url(__dirname, '../../container.html'));
+
+['horizontal', 'vertical'].forEach((orientation) => {
+  test('Splitter icon test', async (t) => {
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]:
```suggestion
  test(`Splitter appearance, orientation='${orientation}'`, async (t) => {
```

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+      dataSource: [{
+        text: 'pane_1',
+      }, {
+        text: 'pane_2',
+      },
+      ],
+    });
+  });
+
+  test('Nested Splitter icon test', async (t) => {
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]:
```suggestion
  test(`Nested Splitter appearance, orientation='${orientation}'`, async (t) => {
```

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts [hunk 8]
```diff
@@ -0,0 +1,62 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import { createWidget } from '../../../helpers/createWidget';
+import { testScreenshot } from '../../../helpers/themeUtils';
+
+fixture.disablePageReloads`Splitter_keyboard`
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]: Let's change fixture name.

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]: correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/276bf53171adf8490925969950bcb18730cb0d5e)

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts (hunk 7 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]: correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/276bf53171adf8490925969950bcb18730cb0d5e)

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts (hunk 6 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]: correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/276bf53171adf8490925969950bcb18730cb0d5e)

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts (hunk 5 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]: correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/276bf53171adf8490925969950bcb18730cb0d5e)

File: packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts (hunk 4 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/testcafe/tests/navigation/splitter/common.ts]: correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/276bf53171adf8490925969950bcb18730cb0d5e)

**Review by Zedwag — DISMISSED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**